docs/guide/faq/README.md

```diff
@@ -35,7 +35,11 @@ This problem can be divided in 2 categories; no logging is shown at all OR inter
 - Try pairing close to a bulb (light) router instead of the coordinator.
 
 ## How do I migrate from one adapter to another?
-Want to migrate from e.g. a CC2531 to a more powerful adapter (e.g. ZZH)? Then follow these instructions:
+
+Want to migrate from e.g. a CC2530/CC2531 to a more powerful adapter (e.g. CC2652/CC1352)? Then follow these instructions below:
+
+Note! Migration requires backup and restore support which is so far only implemented for zstack adapter and deconz adapter. [ezsp adapter](https://github.com/Koenkk/zigbee-herdsman/issues/319) and [zigate adapter](https://github.com/Koenkk/zigbee-herdsman/issues/242) support is experimental so backup and restore has not yet been implemented.
```

Comment:
Deconz also doesn't support backup/restore, only zstack.

Reply:
Does it not? I assumed it did since [hvorragend](https://github.com/hvorragend) wrote that he migrated from an ezsp adapter (Home Assistant SkyConnect) to a deconz adapter (ConBee) -> https://github.com/Koenkk/zigbee2mqtt/discussions/16478#discussioncomment-4970323

Reply:
OK, changed this PR to reflect that neither deconz adapter support backup and restore, as well as opened a directly related feature request for deconz adapter here -> https://github.com/Koenkk/zigbee-herdsman/issues/665